src/modules/filters/result.ts

```diff
@@ -51,7 +52,30 @@ export class FilterResult {
             if (addToResult) {
                 productsResult.push(product);
             }
+
         });
+
+        if (sort !== '') {
+            const sortData = sort.split('/');
+            return sortData[1]! === 'desc'
+                ? this.sortByProperty(productsResult, sortData[0]! as keyof IProduct)
+                : this.sortByProperty(productsResult, sortData[0]! as keyof IProduct).reverse();
+
+        }
         return productsResult;
     }
+
+    private static sortByProperty(productsData: IProduct[], sortProperty: keyof IProduct): IProduct[] {
+        return productsData.sort((a, b) => {
+            const nameA = a[sortProperty];
+            const nameB = b[sortProperty];
```

Comment:
Could you explicit add types here for `a`, `b`, `nameA` and `nameB`. 
Comparison is unclear for the reader.

Reply:
Added.